WPMedia/fusion-news-theme-blocks
# TMEDIA-232 - Blockquote and pullquote formatting

## Description

* Updated Blockquote and Pullquote to use single component to remove duplication
* Updated quote component to output all formatting options that are available to editors in composer
    * Bold
    * Italic
    * Underline
    * Unordered List
    * Ordered List


## Jira Ticket
- [TMEDIA-232](https://arcpublishing.atlassian.net/browse/TMEDIA-232)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-232-blockquote-pullquote-formatting && git remote update --prune origin && git reset --hard origin/canary TMEDIA-232-blockquote-pullquote-formatting`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/article-body-block`
3. Visit article with quotes that have formatting options from composer - http://localhost/2019/09/17/global-kitchen-sink-article/?_website=the-gazette
4. Validate HTML is not output, and quotes are formatted as expected

## Effect Of Changes
### Before

<img width="973" alt="TMEDIA-232-blockquote-before" src="https://user-images.githubusercontent.com/868127/118878236-3de05400-b8e7-11eb-97b6-8b2d1a306f50.png">


<img width="956" alt="TMEDIA-232-pullquote-before" src="https://user-images.githubusercontent.com/868127/118878241-3f118100-b8e7-11eb-97b3-71e028c80941.png">

### After

<img width="956" alt="TMEDIA-232-blockquote-after" src="https://user-images.githubusercontent.com/868127/118878264-46d12580-b8e7-11eb-9c10-13841d02918d.png">


<img width="953" alt="TMEDIA-232-pullquote-after" src="https://user-images.githubusercontent.com/868127/118878267-4769bc00-b8e7-11eb-810c-63f22d623ed6.png">


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
